grpc-server-micrometer/src/main/scala/com/avast/sst/grpc/server/micrometer/MonitoringServerInterceptor.scala

```diff
@@ -24,7 +24,7 @@ class MonitoringServerInterceptor(meterRegistry: MeterRegistry) extends ServerIn
       headers: Metadata,
       next: ServerCallHandler[ReqT, RespT]
   ): ServerCall.Listener[ReqT] = {
-    val prefix = s"grpc.${call.getMethodDescriptor.getFullMethodName}"
+    val prefix = s"grpc.${call.getMethodDescriptor.getFullMethodName.replace('/', '-')}"
```

Comment:
If the only occurrence of `/` as service-method delimiter, maybe it would be better to use `.` instead of `-`, as Micrometer expects `.` as the unified metric separator 🤔

Reply:
I am not sure how many slashes there are in the name so I thought it would be better to replace them with dash and consider that as a single part in the metric name. But both is possible of course.

Reply:
[According to the code](https://github.com/grpc/grpc-java/blob/master/api/src/main/java/io/grpc/MethodDescriptor.java#L379), the `getFullMethodName` returns `com.avast.mypackage/methodName`. Also [this test](https://github.com/grpc/grpc-java/blob/master/api/src/test/java/io/grpc/MethodDescriptorTest.java#L82) says the same.

So I would still prefer to just replace `/` with `.`. It would also make sense to replace the dots in service name (because it includes package) but it is not required IMHO.